includes/class-wc-gateway-stripe.php

```diff
@@ -264,7 +264,7 @@ public function payment_fields() {
 		$display_tokenization = $this->supports( 'tokenization' ) && is_checkout() && $this->saved_cards;
 		$total                = WC()->cart->total;
 		$user_email           = '';
-		$description          = $this->get_description() ? $this->get_description() : '';
+		$description          = ! empty( $this->get_description() ) ? $this->get_description() : '';
```

Comment:
Do we really need this changed? It feels like all the other changes are sufficient.

Reply:
While scanning the code, I just saw it so thought to change it to make it more explicit.